code/game/objects/items/tools/maintenance_tools.dm

```diff
@@ -196,7 +199,9 @@
 /obj/item/tool/weldingtool/Initialize()
 	. = ..()
 	create_reagents(max_fuel)
-	reagents.add_reagent("fuel", max_fuel)
+	if (starting_fuel) {
+		reagents.add_reagent("fuel", max_fuel)
+	}
```

Comment:
```suggestion
	if (starting_fuel)
		reagents.add_reagent("fuel", max_fuel)

```
we don't use braces in our codestyle despite them being valid dm

Reply:
understood